pulp_rpm/app/kickstart/treeinfo.py

```diff
@@ -305,12 +308,14 @@ def variants(self):
 
         for variant_uid in variant_uids:
             variant_key = "variant-" + variant_uid
-            if self._data[variant_key]["repository"] != '.':
+            is_main_repo = self._data[variant_key]["repository"] == DIST_TREE_MAIN_REPO_PATH
+            if is_main_repo:
+                repository = DIST_TREE_MAIN_REPO_PATH
+                packages = PACKAGES_DIRECTORY
+            else:
```

Comment:
No change in logic here.
I had to reread this part many times, and always was confused, so I added the constant in a hope t o make it more readable.